background/navigation-manager.js

```diff
@@ -66,6 +66,21 @@ bgReady.all.then(() => {
       {hostEquals: 'sleazyfork.org', urlMatches},
     ],
   });
+
+  /*
+   * Removes the Get Stylus button on style pages.
+   * Not using manifest.json as adding a content script disables the extension on update.
+   */
+  chrome.webNavigation.onCommitted.addListener(({tabId}) => {
+    chrome.tabs.executeScript(tabId, {
+      file: '/content/install-hook-userstylesworld.js',
+      runAt: 'document_start',
+    });
+  }, {
+    url: [
+      {hostEquals: 'userstyles.world', urlMatches: '/style/\\d+[^/]*/[a-z-0-9]+$'},
```

Comment:
No need for urlMatches because your app is a Single-Page Application so it doesn't produce a real navigation and there will be no onCommitted event, meaning that this won't run if the user first visited the home page and then opened a style.